framework/include/postprocessors/NumFailedTimesteps.h

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Postprocessor.h"
+#include "TimeStepper.h"
+#include "Transient.h"
+#include "GeneralPostprocessor.h"
+//#include "UserObject.h"
+/**
+ * This postprocessor outputs the amount of failed timesteps.
+ */
+class NumFailedTimesteps : public GeneralPostprocessor
```

Comment:
It s TimeStep every other object in the repo, let s be consistent

Reply:
I mean for the object name
the docstring should be written in English, so `time step`